k-distribution/tests/regression-new/checks/concrete.k

```diff
@@ -0,0 +1,26 @@
+module CONCRETE
```

Comment:
Also add a test that does both `concrete(X), symbolic(X), simplification`, because it looks like you added a check for this case too.

Reply:
I did not.